src/fsharp/MethodOverrides.fs

```diff
@@ -520,30 +521,37 @@ module DispatchSlotChecking =
                          if minfo.IsDispatchSlot then
                              yield RequiredSlot(minfo, (*isOptional=*) not minfo.IsAbstract) ]
                 
-                
             // Compute the methods that are available to implement abstract slots from the base class
             //
             // This is used in CheckDispatchSlotsAreImplemented when we think a dispatch slot may not
             // have been implemented. 
             let availPriorOverrides : OverrideInfo list = 
-                if isInterfaceTy g reqdTy then 
+                let isInterface = isInterfaceTy g reqdTy
+                if isInterface && not (g.langVersion.SupportsFeature LanguageFeature.DefaultInterfaceMethodsInterop) then
```

Comment:
Fascinating that the implementation drops out this easily!  Are there cases that this doesn't cover?

Reply:
Tests aren't here yet, but @tihan had a pretty extensive project he was working with to test this ad-hoc and it seemed fairly complete to me.

Reply:
This week I will be working on tests. Thank you @cartermp for getting a draft of the design.

@dsyme , yea, I was happy about how easy it was. Still, we need to get tests to fully verify.